rest/src/main/kotlin/json/request/MessageRequests.kt

```diff
@@ -38,7 +39,7 @@ data class EmbedRequest(
     val type: Optional<String> = Optional.Missing(),
     val description: Optional<String> = Optional.Missing(),
     val url: Optional<String> = Optional.Missing(),
-    val timestamp: Optional<String> = Optional.Missing(),
+    val timestamp: Optional<Instant> = Optional.Missing(),
```

Comment:
Why did this become an instant?

Reply:
Because it can be serialized automatically now I talked about this with @BartArys

Reply:
As mentioned, kx.dt's classes are serializable. Considering the amount of API we're breaking by using kx.dt this seemed like an appropriate change and one that improves the intention of the field. 

This should have been discussed in the PR though instead of on Discord, to avoid the confusion we've run into.

Reply:
That being the case, why didn't we change other Instant feilds from String to Instants; or did I miss something out?
In other words, why is this the only changed field

Reply:
Because this is the only Instant field we send which is an ISO string

Reply:
Members have joined at field 
Synced at for integrations... etc

Reply:
This is specifically about data we're *sending*. The original design decision surround serializing incoming ISO 8601 fields was that it was an unnecessary waste of time to parse a value that in most use-cases won't be read anyhow. This reasoning does not apply to entities we're sending to Discord, since we're forced to encode the data one way or another if it is present.

Reply:
> Members have joined at field
> Synced at for integrations... etc

Carefull hope. Almost everything else doesn't use ISO, but unix timestamps

Reply:
What Bart mentioned was what I was missing
I was wondering why we switched from stringe when we agreed to use strings since theyvare unlikely to be hit